### PR TITLE
add Apache Hudi docs

### PR DESCRIPTION
**Pull request by DanRoscigno**
add docs for Hudi table function and engine

### Changelog category (leave one):

- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/engines/table-engines/integrations/hudi.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Note that the Hudi table must already exist in S3, this command does not take DDL parameters to create a new table.
+
+``` sql
+CREATE TABLE hudi_table
+    ENGINE = Hudi(path, [aws_access_key_id, aws_secret_access_key,])
+```
+
+**Engine parameters**
+
+-   `path` — Bucket url with the path to an existing Hudi table.
```
Comment (kssenii) [inline docs/en/engines/table-engines/integrations/hudi.md:21]: same comment here as for delta lake

**Review by kssenii — APPROVED** [review]

File: docs/en/engines/table-engines/integrations/hudi.md (hunk 1 above)
Comment (DanRoscigno) [inline docs/en/engines/table-engines/integrations/hudi.md:21]:
```suggestion
-   `url` — Bucket url with the path to an existing Hudi table.
```

File: docs/en/sql-reference/table-functions/hudi.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# hudi Table Function
+
+Provides a read-only table-like interface to Apache [Hudi](https://hudi.apache.org/) tables in Amazon S3.
+
+## Syntax
+
+``` sql
+hudi(path [,aws_access_key_id, aws_secret_access_key] [,format] [,structure] [,compression])
```
Comment (DanRoscigno) [inline docs/en/sql-reference/table-functions/hudi.md:13]:
```suggestion
hudi(url [,aws_access_key_id, aws_secret_access_key] [,format] [,structure] [,compression])
```

File: docs/en/sql-reference/table-functions/hudi.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Syntax
+
+``` sql
+hudi(url [,aws_access_key_id, aws_secret_access_key] [,format] [,structure] [,compression])
+```
+
+## Arguments
+
+- `path` — Bucket url with the path to an existing Hudi table in S3.
```
Comment (DanRoscigno) [inline docs/en/sql-reference/table-functions/hudi.md:18]:
```suggestion
- `url` — Bucket url with the path to an existing Hudi table in S3.
```

File: docs/en/engines/table-engines/integrations/hudi.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+This engine provides a read-only integration with existing Apache [Hudi](https://hudi.apache.org/) tables in Amazon S3.
+
+## Create Table
+
+Note that the Hudi table must already exist in S3, this command does not take DDL parameters to create a new table.
+
+``` sql
+CREATE TABLE hudi_table
+    ENGINE = Hudi(path, [aws_access_key_id, aws_secret_access_key,])
```
Comment (DanRoscigno) [inline docs/en/engines/table-engines/integrations/hudi.md:16]:
```suggestion
    ENGINE = Hudi(url, [aws_access_key_id, aws_secret_access_key,])
```

**Pull request merged**